docker/swift-t/install-swift-t.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -eux
+
+# INSTALL SWIFT/T
+
+# Setup
+# PATH=/usr/lib64/mpich/bin:$PATH
+PATH=/usr/lib64/openmpi/bin:$PATH
+PATH=/opt/tcl-8.6.11/bin:$PATH
```

Comment:
Do these need to be set at runtime for `swift-t` to work?  If so, they can be added to the Dockerfile with `ENV` and will be set at runtime when the container is launched.  If not and these just need to be set for building, then ignore this comment :)

Reply:
These are just for build time.